app/models/viewmodels/CasesTabViewModel.scala

```diff
@@ -111,6 +111,25 @@ object CasesTabViewModel {
     )
   }
 
+  def liabilityCases(allQueueCases: Seq[Case]): CasesTabViewModel = {
+
+    val actCases = allQueueCases.filter(c => c.queueId.contains(Queues.act.id) && c.application.isLiabilityOrder)
+    val capCases = allQueueCases.filter(c => c.queueId.contains(Queues.cap.id) && c.application.isLiabilityOrder)
+    val carsCases = allQueueCases.filter(c => c.queueId.contains(Queues.cars.id) && c.application.isLiabilityOrder)
+    val elmCases = allQueueCases.filter(c => c.queueId.contains(Queues.elm.id) && c.application.isLiabilityOrder)
```

Comment:
I think we could filter out liability cases first so we aren't repeatedly filtering the whole list:
```suggestion
    val liabilityCases = allQueueCases.filter(_.application.isLiabilityOrder)
    val actCases = liabilityCases.filter(c => c.queueId.contains(Queues.act.id))
    val capCases = liabilityCases.filter(c => c.queueId.contains(Queues.cap.id))
    val carsCases = liabilityCases.filter(c => c.queueId.contains(Queues.cars.id))
    val elmCases = liabilityCases.filter(c => c.queueId.contains(Queues.elm.id))
```